account_payment_order/models/account_payment_order.py

```diff
@@ -330,7 +330,7 @@ def generated2uploaded(self):
         return True
 
     @api.multi
-    def _prepare_move(self):
+    def _prepare_move(self, bank_lines=None):
```

Comment:
No, I don't want to change the bank payment lines. I want to compute `ref` from the bank payment lines.